21/README.md

```diff
@@ -41,41 +41,33 @@ Consider a network of 4 nodes `{A,B,C,D}`. If a node `A` wishes to add a new nod
 
-3. The node operator can list the contents of `election_id` request using,
+3. The node operator can list the contents of `election_id` request using [`bigchaindb election show ...`](../18),
 
   ```
-  $ bigchaindb upsert-validator show <election_id>
+  $ bigchaindb election show <election_id>
   public_key=Wn2DedV9OA0LJJjOxr7Sl7jqCSYjQihA6dCBX+iHaEI=
   power=10
   node_id=82190eb6396bdd80b83aef0f931d0f45738ed075
+  status=ONGOING
   ```
 
-  The above command list the details about the node which is being added/updated/deleted from the network.
-
 4. If the node operator aggrees to the operation being proposed by the `election_id` then they can vote on the same using the following,
 
   ```
-  $ bigchaindb upsert-validator approve <election_id> --private-key /home/user/.tendermint/config/priv_validator.json
+  $ bigchaindb election approve <election_id> --private-key /home/user/.tendermint/config/priv_validator.json
   ```
 
-  The above command `POST`s a [`VALIDATOR_ELECTION_VOTE`][spec_validator_election_vote] transaction casting the vote of the node for given `election_id`.
-  NOTE: The `VALIDATOR_ELECTION_VOTE` transaction is signed using the private key generated and stored by Tendermint in `priv_validator.json`.
-
-5. Node operators can check the status of the election as follows,
+  The above command `POST`s a [`ELECTION_VOTE`][spec_election_vote] transaction casting the vote of the node for given `election_id`.
+  NOTE: The `ELECTION_VOTE` transaction is signed using the private key generated and stored by Tendermint in `priv_validator.json`.
 
-  ```
-  $ bigchaindb upsert-validator status <election_id>
-  votes_recieved=<Sum_of_votes_recieved>
-  votes_allocated=<Sum_of_votes_allocated_in_election>
-  network_size=<Total_network_power>
-  ```
 
 ### Validating transaction schemas
-Two new transaction specs namely, [`VALIDATOR_ELECTION`][spec_validator_election] and [`VALIDATOR_ELECTION_VOTE`][spec_validator_election_vote] have been proposed for implementing this BEP. It is worth mentioning that `VALIDATOR_ELECTION` is an extension of [`CREATE` transaction](../13) and `VALIDATOR_ELECTION_VOTE` is an extension of [`TRANSFER` transaction](../13). Consequentially the validation of each of these new transaction specs must be evaluated by re-using the validation for the base spec after which the schema can be validated against its extended spec.
+Two new transaction specs namely, [`VALIDATOR_ELECTION`][spec_validator_election] and [`ELECTION_VOTE`][spec_validator_election_vote] have been proposed for implementing this BEP. It is worth mentioning that `VALIDATOR_ELECTION` is an extension of [`CREATE` transaction](../13) and `ELECTION_VOTE` is an extension of [`TRANSFER` transaction](../13). Consequentially the validation of each of these new transaction specs must be evaluated by re-using the validation for the base spec after which the schema can be validated against its extended spec.
```

Comment:
Only one new spec is proposed in this BEP, not two.

Reply:
There seems to be some confusion about what a "transaction spec" is. There have only been two transaction specs in the whole history of BigchainDB: v1 and v2, defined in BEP-12 and BEP-13 respectively.

Here, I think what was meant is that two new _types_ of transaction are proposed (under the umbrella of the v2 transaction spec).

I will update this PR accordingly.